libiocage/cli/__init__.py

```diff
@@ -85,9 +85,10 @@ def get_command(self, ctx, name):
                 if mod.__rootcmd__ and "--help" not in sys.argv[1:]:
                     if len(sys.argv) != 1:
                         if os.geteuid() != 0:
+                            app_name = mod.__name__.rsplit(".")[-1]
                             logger.error(
-                                f"You need to have root privileges"
-                                f" to run {mod.__name__}"
+                                "You need to have root privileges"
+                                f" to run {app_name}"
```

Comment:
@skarekrow Moved this to a more descriptive variable that prevents quotation with single-quotes.